README.md

```diff
@@ -0,0 +1,31 @@
+Kuali Rice
+==========
+Kuali Rice is an application development framework and middleware suite developed for and by the higher education community.  Kuali Rice is being used by several Kuali universiy enterprise applications, for its build in eDocLite document routing, and as basis for custom applications.
```

Comment:
Oh, I see what you mean, the eDocLite routing functionality is built in.  I was just reading it wrong, I don't think the dash is needed.